scripts/minimum_deps

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+"""
+Generate a requirements-min.txt file based on install_requres
+"""
+import pkg_resources
+import urllib.request
+import json
+import warnings
+
+
+def get_minimum_version(requirement):
+    """Return minimum version available on PyPi for a given version specification"""
+    content = urllib.request.urlopen(f'https://pypi.python.org/pypi/{requirement.name}/json').read()
+    versions = sorted([pkg_resources.parse_version(v) for v in json.loads(content)["releases"].keys()])
+    # print(requirement)
+    # print(versions)
+    for version in versions:
+        if version in requirement:
+            return version
+    # If there was no version specifier in setup.py, return the 2nd lowest
+    # version available on PyPi, avoiding 0.0 versions
```

Comment:
I looked a few packages and discovered that the `0` version is not consistent.  The earliest version of `asdf` is `0`, but for `scipy` it's `0.4.4`.

Reply:
Right, I saw that too.